api/control/v1alpha1/types.go

```diff
@@ -161,37 +161,37 @@ type DownsyncObjectTest struct {
 
 // +kubebuilder:object:root=true
 
-// PlacementList contains a list of Placement
-type PlacementList struct {
+// BindingPolicyList contains a list of BindingPolicies
+type BindingPolicyList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
-	Items           []Placement `json:"items"`
+	Items           []BindingPolicy `json:"items"`
 }
 
-// PlacementDecision is mapped 1:1 to a single Placement object.
-// The decision resource reflects the resolution of the Placement's selectors,
+// Binding is mapped 1:1 to a single BindingPolicy object.
+// The decision resource reflects the resolution of the BindingPolicy's selectors,
 // and explicitly reflects which objects should go to what destinations.
 //
 // +genclient
 // +genclient:nonNamespaced
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:scope=Cluster,shortName={pd}
-type PlacementDecision struct {
+// +kubebuilder:resource:scope=Cluster,shortName={bd}
```

Comment:
FYI: "bd" is not bad. I could read it as "Binding Decision", and that would warm my heart, until I remember that is not what it means. If the full name is "Binding" then I think "bdg" would be a better shortname.

Reply:
Agreed and changed.